src/common/mfu_flist.c

```diff
@@ -433,10 +454,12 @@ static void list_compute_summary(flist_t* flist)
     uint64_t max_name = 0;
     elem_t* current = flist->list_head;
     while (current != NULL) {
-        uint64_t len = (uint64_t)(strlen(current->file) + 1);
-        if (len > max_name) {
-            max_name = len;
-        }
+        if (current->file != NULL) {
+            uint64_t len = (uint64_t)(strlen(current->file) + 1);
+            if (len > max_name) {
+                max_name = len;
+            }
+	}
```

Comment:
@dsikich Just a thought... I wonder if it could be feasible/useful to convert the oid to a string and use that as the name? This could tie into what we discussed before about being able to write oids to a file, similar to how posix filenames are written to a file in dwalk.
We can address this later when it becomes relevant, but something to keep in mind.

Reply:
@daltonbohning I think we could. I was going to try and do this with the serialization but it ended up not being necessary. I don't like that every time you go to do a key lookup you will have to decode the string (the whole list of strings I imagine after reading it in). I think trying this would be a good idea though so that we can try out using libcircle, and see if it helps performance.

Reply:
@dsikich Yeah, we can worry about this separately when we look at performance, etc.